src/role-helpers.js

```diff
@@ -198,6 +198,9 @@ function computeAriaChecked(element) {
   // implicit value from html-aam mappings: https://www.w3.org/TR/html-aam-1.0/#html-attribute-state-and-property-mappings
   // https://www.w3.org/TR/html-aam-1.0/#details-id-56
   // https://www.w3.org/TR/html-aam-1.0/#details-id-67
+  if ('indeterminate' in element && element.indeterminate) {
+    return false
```

Comment:
This seems confusing to me. A mixed checkbox is neither check nor unchecked. Right now we consider it unchecked as far as I can tell.

Reply:
I agree that it's confusing, but that's the spec.
A checkbox in indeterminate state is neither checked not unchecked.
What do you suggest?

Reply:
Sorry, missed the second part of your comment.
You're right, I had a mistake in the test.
Pushing a fix

Reply:
It's fine to not be able to query mixed checkboxes for now. But returning indeterminate ones for `checked: false` would be incorrect.